src/kontrol/prove.py

```diff
@@ -480,6 +480,10 @@ def _method_to_cfg(
         init_node = cfg.create_node(init_cterm)
         new_node_ids = [init_node.id]
         init_node_id = init_node.id
+        if not method.is_test:
+            _LOGGER.info(
+                f'{contract.name_with_path}.{method.signature} is not prefixed with test or testFail, therefore it is not reported as failing in the presence of reverts or assertion violations.'
```

Comment:
Thanks for updating! I'd also suggest adding other test prefixes, i.e., 
```suggestion
                f'{contract.name_with_path}.{method.signature} is not prefixed with 'test', 'prove', or 'check', therefore it is not reported as failing in the presence of reverts or assertion violations.'
```